src/go/rpk/pkg/cli/cmd/debug/bundle.go

```diff
@@ -0,0 +1,552 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package debug
+
+import (
+	"archive/zip"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"sort"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/Shopify/sarama"
+	"github.com/hashicorp/go-multierror"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/common"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/system"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/system/syslog"
+	"gopkg.in/yaml.v2"
+)
+
+// Use the same date specs as journalctl (see `man journalctl`).
+const timeHelpText = `Date specifications should be of the format "2012-10-30
```

Comment:
This help text is way too long to be displayed alongside a flag, I think we can just say `(journalctl date format; e.g. YYYY-MM-DD)`. We may want to extend the since / until to more than just journalctl down the line, but I think this is fine to start since we expect debug to be used by us (so if we extend the format later ... we break ourselves).

Reply:
Fixed.